TACC/Core-CMS-Custom
# bug/WP-741: Missed bugs follow on

## Overview
Fixes some issues missed in first merge of WP-741 PR
…

## Related

[WP-741 PR](https://github.com/TACC/Core-CMS-Custom/pull/358)
…

## Changes
- Registration form view get() now uses directly-imported db util functions (`get_registrations`, `get_registration_entities`, `get_registration_contacts`)
- Spacing for Add/Remove entities and contacts wrapping + styling fix
- State field now has initial value corresponding to default choice + initial touched in event user does not interact with this field
- Entity + Contact headers (per entry) now match prod
…

## Testing

1. Go to http://localhost:8000/register/request-to-submit/ and confirm styling matches prod (txapcd.org/register/request-to-submit)
2. Fill out form without touching state drop down and confirm Alabama shows up in registration listing upon successful submission

## UI

…
